components/LabelWithValue.js

```diff
@@ -35,14 +35,14 @@ const LabelWithValue = (props) => {
           : styles.labelWithValueContainer
       } ${className}`}
     >
-      <span className={styles.label}>{label}</span>
+      {label && <span className={styles.label}>{label}</span>}
```

Comment:
Not sure what to call it exactly, but `<LabelWithValue />` doesn't quite feel right if it may not actually render a label

Reply:
If I have time, I'll go back through and create new components for just `Label` and `Value`.

Reply:
Made this [issue](https://github.com/rode/rode-ui/issues/144) in case I do not get to it in time.